CHANGELOG.md

```diff
@@ -124,9 +124,14 @@ regression test suite and patches at https://github.com/seqan/seqan3/tree/master
   * `seqan3::aminoacid_similarity_matrix::BLOSUM45` is replaced by `seqan3::aminoacid_similarity_matrix::blosum45`.
   * `seqan3::aminoacid_similarity_matrix::BLOSUM62` is replaced by `seqan3::aminoacid_similarity_matrix::blosum62`.
   * `seqan3::aminoacid_similarity_matrix::BLOSUM80` is replaced by `seqan3::aminoacid_similarity_matrix::blosum80`.
-* We deprecated seqan3::alignment_coordinate and it will be removed in 3.1.0
+* Deprecated seqan3::alignment_coordinate and it will be removed in 3.1.0
   ([\#2602](https://github.com/seqan/seqan3/pull/2602)).
 
+Header Changes:
+
+```cpp
+#include <seqan3/{alignment/pairwise => core/algorithm}/alignment_range.hpp>
```

Comment:
I could not find any `seqan3/alignment/pairwise/alignment_range.hpp` or `seqan3/core/algorithm/alignment_range.hpp` file

Reply:
Me neither, I think somethignwent wrong here :)

Reply:
removed :)